### PR TITLE
Use basic auth instead of JWT in tests.

### PR DESCRIPTION
**Pull request by imeredith**
# Description
@vivek seeing occasionally flaky tests that I believe are due to JWT not setting the user properly. Will be documenting it in a ticket for you to take a look at

See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/ProfileApiTest.java [hunk 1]
```diff
@@ -365,7 +365,7 @@ public void badTokenTest1() throws Exception {
     }
 
 
-    @Test
+    //@Test
```
Comment (cliffmeyers) [inline blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/ProfileApiTest.java]: Should `@Ignore` be used so we can see in reports that they are actually skipped?

**Comment by imeredith** [comment]
I am trying that but i dont see how that can fix anything given we set that prop in BaseTest
```
  public BaseTest() {
        System.setProperty("BLUEOCEAN_FEATURE_JWT_AUTHENTICATION", "true");
        j = new JenkinsRule();
    }
```

**Comment by vivek** [comment]
@imeredith thats inside constructor, static blocks are loaded during class load time and much before JenkinsRule or anything else gets setup. I think it's worth a shot.

**Comment by imeredith** [comment]
@vivek That static block thing didnt work. Merging this for now because it is blocking other people from getting things done. Can we add them back in if we figure it out.

**Comment by vivek** [comment]
@imeredith 👍

**Comment by i386** [comment]
Cherry picked to release branch

**Pull request merged**